### PR TITLE
Point to the latest GPLOT `support/HAFS` branch

### PR DESCRIPTION
**Pull request by ghassan-alaka**
## Description of changes
Update the GPLOT submodule (`hafs_graphics.fd/hrd_gplot`) and .gitmodules file to point to the latest `release/0.4.0` branch. The branch is forked here:  https://github.com/hafs-community/GPLOT/tree/release/0.4.0

## Contributors (optional)
@lgramer @AndrewHazelton 

## Tests conducted
Extensive testing on Orion, Hera, and Jet.

## Application-level regression test status
Running the HAFS application-level regression tests is currently performed by code reviewers after the developer creates the initial PR. As regression tests are conducted, the testers should use the checklist below to indicate **successful** regression tests. You may add other tests as needed. If a test fails, do not check the box. Instead, describe the failure in the PR comments, noting the platform where the test failed.

- [ ] Jet
- [ ] Hera
- [ ] Orion
- [ ] WCOSS2

### CODE REVIEW

**Comment by BinLiu-NOAA** [comment]
@ghassan-alaka, just quickly check with you, do you meant to merge this PR back to HAFS develop branch or the support/hafs.v1.1.0 branch (the real-time parallel experiment branch)?

**Comment by ghassan-alaka** [comment]
This is a good question for @AndrewHazelton . It should definitely go into `develop` to carry it forward to future developments. However, if Andy is using `support/hafs.v1.1.0` then these updates should also be merged into that branch as well.

**Comment by AndrewHazelton** [comment]
Right, I think it would be good to have it in both.

File: .gitmodules [hunk 1]
```diff
@@ -17,7 +17,7 @@
 [submodule "GPLOT"]
     path = sorc/hafs_graphics.fd/hrd_gplot
     url = https://github.com/hafs-community/GPLOT.git
-    branch = support/HAFS
+    branch = release/0.4.0
```
Comment (BinLiu-NOAA) [inline .gitmodules]: @ghassan-alaka, since the HAFS develop branch typically has its submodules point to the support/HAFS branch or the corresponding authorized main or develop branch, could you please merge the release/0.4.0 branch into the support/HAFS branch of https://github.com/hafs-community/GPLOT.git. After that, you can revert this change to point to the support/HAFS branch here. Thanks!

Comment (ghassan-alaka) [reply .gitmodules]: OK sounds good @BinLiu-NOAA . I just updated the `support/HAFS` branch within the hafs-community/GPLOT repo.

**Comment by ghassan-alaka** [comment]
@BinLiu-NOAA The .gitmodules file has been updated. It now points to the `support/HAFS` branch of the GPLOT submodule.

**Comment by BinLiu-NOAA** [comment]
> @BinLiu-NOAA The .gitmodules file has been updated. It now points to the `support/HAFS` branch of the GPLOT submodule.

Thanks, @ghassan-alaka! One more thing, could you please also update the submodule to point the latest commit of the support/HAFS branch?
https://github.com/hafs-community/GPLOT/commit/f1d8e2671d7ad0ea229596b5c7541538d0135ba9

**Comment by BinLiu-NOAA** [comment]
Any way, I have just updated to point to the latest commit of the support/HAFS branch (https://github.com/hafs-community/GPLOT/commit/f1d8e2671d7ad0ea229596b5c7541538d0135ba9)

**Review by BinLiu-NOAA — APPROVED** [review]
Looks good to me.

Since this is just a submodule updates for hafs_graphics.fd/hrd_gplot and its not tested by any regression tests, I think we can skip the HAFS application/workflow level regression tests for this PR.

**Review by AndrewHazelton — APPROVED** [review]
This looks good to me!
